src/freenet/support/WrapperKeepalive.java

```diff
@@ -14,7 +15,7 @@
   public void run() {
     while (!shutdown) {
       try {
-        WrapperManager.signalStarting(INTERVAL);
+        WrapperManager.signalStarting(INTERVAL + (int)SECONDS.toMillis(5));
```

Comment:
5 seconds is to account for delay in the thread wakeup? What's the reasoning for that value? Is there a more reliable way to schedule this to be regularly called that would avoid the need for a fudge factor like this?